Formula/e/erlang.rb

```diff
@@ -79,15 +84,6 @@ def install
     doc.install resource("html")
   end
 
-  def caveats
-    <<~EOS
-      Man pages can be found in:
-        #{opt_lib}/erlang/man
-
-      Access them with `erl -man`, or add this directory to MANPATH.
-    EOS
-  end
```

Comment:
why removing this?

Reply:
There's no more manpages, for 27.0.1. They might come back, later, but then the formula will have to be updated for it.

Reply:
My initial commit had more stuff in the body, but because I kept running into "Commit style" issues I was tweaking here and there, and eventually must've removed the body in desperation 😄

Reply:
I see, make sense. 👍